tests/typecheck/managers/querysets/test_values_list.yml

```diff
@@ -26,6 +26,25 @@
                     name = models.CharField(max_length=100)
                     age = models.IntegerField()
 
+-   case: values_list_field_repetition
+    main: |
+        from myapp.models import MyUser
+
+        # Values tuples can have the same field repeated
+        values_tuple = MyUser.objects.values_list('name', 'name').get()
+        reveal_type(values_tuple)  # N: Revealed type is "Tuple[builtins.str, builtins.str]"
+        reveal_type(values_tuple[0])  # N: Revealed type is "builtins.str"
+        reveal_type(values_tuple[1])  # N: Revealed type is "builtins.str"
+    installed_apps:
+        - myapp
+    files:
+        -   path: myapp/__init__.py
+        -   path: myapp/models.py
+            content: |
+                from django.db import models
+                class MyUser(models.Model):
+                    name = models.CharField(max_length=100)
```

Comment:
Can we please have another field for this test? Something like `age = models.IntegerField`

Reply:
Done - good suggestion!